res/contribute.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
According to the docs it should be at the project root?

Reply:
It's copied into `webapp` in the build [here](https://github.com/vector-im/riot-web/pull/12251/files#diff-c1e2b8b85603326af7b040e66f254aefR69) and as such is in the same place as `config.json` if I've understood the build process correctly?

Reply:
This is more what I'm referencing:
![image](https://user-images.githubusercontent.com/1190097/73848294-c2f14800-481f-11ea-9b01-030d883fce80.png)

Reply:
the build tools can pull from there too, though it'll also need adding to the `files` in the package.json

Reply:
> This is more what I'm referencing:
> ![image](https://user-images.githubusercontent.com/1190097/73848294-c2f14800-481f-11ea-9b01-030d883fce80.png)

Ah right, you're referring to how it's deployed on riot.im/app for example. True, this wont work on the NV maintained riot web's at all. I guess the idea was not so much the NV installed riot but any hosted riot on a root path of their domain.

Reply:
> the build tools can pull from there too, though it'll also need adding to the `files` in the package.json

`res` is already in files - will that do? I verified `yarn build` does end up copying the file, but if it's needed explicitly somewhere else can add.

Reply:
The problem has nothing to do with the build process: I literally mean move this file up one directory.

Reply:
Sorry, I seriously have had enough coffee but somehow managed to totally misunderstand everything you clearly wrote while juggling other things. Fixed now.

Reply:
No worries, thanks!